tests/test_package_manager/test_v0.py

```diff
@@ -0,0 +1,188 @@
+# -*- coding: utf-8 -*-
+# ------------------------------------------------------------------------------
+#
+#   Copyright 2022 Valory AG
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+# ------------------------------------------------------------------------------
+
+"""Test package manager base."""
+
+
+import json
+import logging
+from collections import OrderedDict
+from pathlib import Path
+from unittest import mock
+
+from aea.package_manager.v0 import PackageManagerV0
+from aea.test_tools.test_cases import BaseAEATestCase
+
+from tests.test_package_manager.test_base import (
+    DUMMY_PACKAGE_HASH,
+    DUMMY_PACKAGE_ID,
+    EXAMPLE_PACKAGE_HASH,
+    EXAMPLE_PACKAGE_ID,
+    PACKAGE_JSON_FILE,
+)
+
+
+class TestPackageManagerV0(BaseAEATestCase):
+    """Test aea package manager."""
+
+    use_packages_dir: bool = True
+    packages_json_file: Path
+
+    def setup(
+        self,
+    ) -> None:
+        """Setup test."""
+        packages_v1 = json.loads(PACKAGE_JSON_FILE.read_text(encoding="utf-8"))
+
+        self.packages_json_file = self.packages_dir_path / "packages.json"
+        self.packages_json_file.write_text(
+            json.dumps({**packages_v1["dev"], **packages_v1["third_party"]})
+        )
```

Comment:
comment in the code explaining why we have `packages_v1` in the `PackageManagerV0` test would be useful